integrations/GoogleAds/browser.js

```diff
@@ -83,21 +88,34 @@ class GoogleAds {
         window.gtag("event", eventName, properties);
       }
     } else {
-      const { event } = rudderElement.message;
+      let { event } = rudderElement.message;
```

Comment:
Why was the 'const' qualifier changed?

Reply:
This was changed as per the earlier implementation, reverting it to const.